lib/util.ts

```diff
@@ -174,7 +174,7 @@ export const osend = function(os: WriteStreamMinimal) {
   });
 }
 
-export const osopen = function(path: string | Buffer | URL, opts?: string | WriterOptions): Promise<WriteStream> {
+export const osopen = function(path: string | Buffer | URL, opts?: WriterOptions): Promise<WriteStream> {
```

Comment:
removed `string` as a type because the only thing that calls osopen can never pass a string.